manifests/kustomize/env/platform-agnostic-multi-user/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 
 bases:
   - ../../base/installs/multi-user
+  - ../../base/metadata/overlays/db
```

Comment:
May I confirm your plan with this overlay?
I'd prefer only keeping it for backward compatibility reasons, because of the unnecessary burden of maintaining two DBs. Most distributions should configure their own overlay of the db using managed DB services for both KFP and Metadata.

Therefore, my suggestion would be: split into two `platform-agnostic-multi-user` envs
* One has name `platform-agnostic-multi-user-legacy` which uses this db overlay.
* While the other has name `platform-agnostic-multi-user` which keeps using the base metadata.

What do you think?

Reply:
This is a great suggestion! I implemented it.